src/app/components/InputField.svelte

```diff
@@ -9,19 +9,27 @@
   export let size: string | undefined = "";
   export let cyData: string | undefined = "";
   export let id: string;
-  export let value: any;
   export let subtext = "";
   export let error = "";
   export let placeholder = "";
-
+  let inputValue: string = "";
   const dispatch = createEventDispatcher();
 
   function dispatchInputValue(event: Event) {
     const target = event.target as HTMLTextAreaElement;
     const cleanId = target.id.split("-").pop();
     dispatch("message", {
       key: cleanId,
-      value: value,
+      value: inputValue,
+    });
+  }
+
+  function dispatchInputValueOnInput(event: Event) {
+    const target = event.target as HTMLTextAreaElement;
+    const cleanId = target.id.split("-").pop();
+    dispatch("messageOnChange", {
```

Comment:
I wonder if there's a better name for this event. The camel case is weird in the message name, since I don't think many DOMEvent names have camel case at all.
https://developer.mozilla.org/en-US/docs/Web/Events

Distressingly, the most common event name convention is `abunchofsmooshedtogetherwords` 😩 

Maybe...

```suggestion
    dispatch("keytyped", {
```